homeassistant/components/zha/core/registries.py

```diff
@@ -110,6 +112,7 @@
         zigpy.profiles.zha.DeviceType.SHADE: COVER,
         zigpy.profiles.zha.DeviceType.SMART_PLUG: SWITCH,
         zigpy.profiles.zha.DeviceType.IAS_ANCILLARY_CONTROL: ALARM,
+        zigpy.profiles.zha.DeviceType.SIMPLE_SENSOR: BINARY_SENSOR,
```

Comment:
Remove it from the DEVICE_CLASS. It is better for to handle this as a single_cluster_device.